### ISSUE TITLE
Fatal error with SocialConnection ressource parsing

### ISSUE THREAD

**Issue by Grestorn** [Patreon/patreon-java#19]
Since recently (maybe even today), fetchAllPledges is failing with this exception: 

```
Unrecognized field "url" (class com.patreon.resources.shared.SocialConnections$UserIdObject), not marked as ignorable (2 known properties: "user_id", "scopes"])
 at [Source: N/A; line: -1, column: -1] (through reference chain: com.patreon.resources.User["social_connections"]->com.patreon.resources.shared.SocialConnections["discord"]->com.patreon.resources.shared.SocialConnections$UserIdObject["url"])
```

The campaign id is 1952686

**Comment by nperickson** [comment]
Hey there,

Would you be able to send us an email at platform@patreon.com? A member of the team can help assist with this.

**Comment by vlanard** [comment]
Hi @Grestorn, have you been able to see this resolved?

**Comment by Grestorn** [comment]
Yes, at first I fixed it myself but then they've updated to 0.4.1 which also fixed it.

Do you still experience the problem?

**Comment by vlanard** [comment]
Nope, just wanted to follow up, thanks!
